python/lsst/ip/isr/photodiode.py

```diff
@@ -169,15 +169,38 @@ def fromTable(cls, tableList):
             The calibration defined in the tables.
         """
         dataTable = tableList[0]
-
         metadata = dataTable.meta
+
+        # Dump useless entries that are carried over from merging
+        # HDU[0]'s header with the header from HDU[1] (which has the
+        # data table).
+        for key in ("SIMPLE", "BITPIX", "NAXIS", "EXTEND"):
+            metadata.pop(key)
+
+        # Do translations:
+        instrument = metadata.pop("INSTRUME", "NO_INSTRUMENT")
+        location = metadata.pop("LOCATN", "NO_LOCATION")
+
+        if instrument == "Electrometer_index_201" and location == "AuxTel":
+            metadata["INSTRUME"] = "LATISS"
+        elif instrument == "Electrometer_index_101" and location == "MainTel":
+            metadata["INSTRUME"] = "LSSTCam"
+        else:
+            # This will cause problems in ingest, but we don't know
+            # what to associate it with.
+            metadata["INSTRUME"] = "Unknown"
```

Comment:
Is this really the correct location for this code?  (Though I'm not entirely sure where else).  And does this create any problems with the old photodiode data from Run7?

Reply:
It's was the least bad place when I was thinking it through, and since we only have photodiode/electrometer data for LSSTCam.  I've tried to keep the ingest code able to accept the older versions, but I don't think those populated `INSTRUME`.  I have changed this to default to `None` on the initial `pop`, and use that value here, whatever it is, if we couldn't determine the instrument from the if/elses.